test/Constraints/result_builder_diags.swift

```diff
@@ -443,8 +443,8 @@ func getSomeEnumOverloaded(_: Int) -> E2 { return .b(0, nil) }
 func testOverloadedSwitch() {
   tuplify(true) { c in
     // FIXME: Bad source location.
```

Comment:
Does this comment still make sense now that source loc seems to be correct?

Reply:
Ah yeah, that doesn't make sense anymore!

Reply:
Removed.